tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs

```diff
@@ -78,7 +91,7 @@ private async Task BrowseButtonClick(object sender)
         }
     }
 
-    private void MoveDirectory(string sourceDirectory, string targetDirectory)
+    private bool MoveDirectory(string sourceDirectory, string targetDirectory)

```

Comment:
I'm on the fence about returning `bool` here.  Line `172` does not do anything different if the move failed.  In this context  the caller can assume the move finished successfully if `MoveDirectory` didn't throw.

On the other, maybe other parts of the code branch depending on the result.

I'll leave it up to you.  C# is a "If it does not throw, we're good"